packages/charts/src/components/CarbonChart/CarbonChart.test.js

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import { CarbonChart, CarbonChartType } from '.';
+import { render } from '@testing-library/react';
+
+const TEXT = 'Test Text';
+const dataSet = [
+  {
+    group: 'Dataset 1',
+    date: '2019-01-01T05:00:00.000Z',
+    value: 0,
+  },
+  {
+    group: 'Dataset 2',
+    date: '2019-01-01T05:00:00.000Z',
+    value: 47263,
+  },
+];
+
+describe.skip('CarbonChart', () => {
```

Comment:
What happens if we remove this skip?

Reply:
'Element type is invalid: expected a string (for built-in components) or a class/function (for composite components) but got: undefined. You likely forgot to export your component from the file it's defined in, or you might have mixed up default and named imports.'

Reply:
Sounds like a legitimate thing we should address